Dockerfile

```diff
@@ -27,29 +30,32 @@ RUN lerna run --scope @nci-gdc/sapien build
 RUN lerna run --scope portal-proto build
 # ==================================================================
 
-FROM node:20.11.0-alpine3.18 AS runner
+FROM ${REGISTRY}/nodejs20:${BASE_VERSION} AS runner
+ARG NAME=gdc-frontend-framework
+
+LABEL org.opencontainers.image.title=${NAME} \
+      org.opencontainers.image.description="${NAME} container image" \
+      org.opencontainers.image.source="https://github.com/NCI-GDC/${NAME}" \
+      org.opencontainers.image.vendor="NCI GDC"
+
 WORKDIR /app
 ENV NODE_ENV=production \
     PORT=3000
 
-RUN  addgroup --system --gid 1001 nextjs && adduser --system --uid 1001 nextjs
-
-COPY --from=builder --chown=nextjs:nextjs /app/lerna.json ./lerna.json
-COPY --from=builder --chown=nextjs:nextjs /app/package.json ./package.json
-COPY --from=builder --chown=nextjs:nextjs /app/node_modules ./node_modules
-COPY --from=builder --chown=nextjs:nextjs /app/packages/portal-proto/public ./packages/portal-proto/public
-COPY --from=builder --chown=nextjs:nextjs /app/packages/portal-proto/package.json ./packages/portal-proto/package.json
-COPY --from=builder --chown=nextjs:nextjs /app/packages/portal-proto/.next ./packages/portal-proto/.next
-COPY --from=builder --chown=nextjs:nextjs /app/packages/portal-proto/node_modules ./packages/portal-proto/node_modules
-COPY --from=builder --chown=nextjs:nextjs /app/packages/portal-proto/next.config.js ./packages/portal-proto/next.config.js
+COPY --from=builder --chown=app:app /app/lerna.json ./lerna.json
+COPY --from=builder --chown=app:app /app/package.json ./package.json
+COPY --from=builder --chown=app:app /app/node_modules ./node_modules
+COPY --from=builder --chown=app:app /app/packages/portal-proto/public ./packages/portal-proto/public
+COPY --from=builder --chown=app:app /app/packages/portal-proto/package.json ./packages/portal-proto/package.json
+COPY --from=builder --chown=app:app /app/packages/portal-proto/.next ./packages/portal-proto/.next
+COPY --from=builder --chown=app:app /app/packages/portal-proto/node_modules ./packages/portal-proto/node_modules
+COPY --from=builder --chown=app:app /app/packages/portal-proto/next.config.js ./packages/portal-proto/next.config.js
 
 RUN mkdir -p ./packages/portal-proto/.next \
-  && chown nextjs:nextjs ./packages/portal-proto/.next
+  && chown app:app ./packages/portal-proto/.next
```

Comment:
so UID:GID app:app exists at this point?

Without being added? Or is it part of the base image already?

Reply:
yes